.pylintdict

```diff
@@ -96,6 +98,8 @@ otimes
 ovo
 ovr
 param
+parameterize
+parameterized
```

Comment:
I get that 'paramterize(d)` is valid too - but I think we have usually tried to standardize to the one way, which is  'parametrized` without the e after the t as you see below here.